tests/test_schema_errors/test_bad_type.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from openapi_tester import OpenAPISchemaError, SchemaTester
+
+example_schema_array = {"type": "array", "items": {"type": "string"}}
+example_array = ["string"]
+
+tester = SchemaTester()
+
+
+def test_bad_schema_type():
+    """
+    Unsupported schema types should raise an error.
+    """
+    for _type in ["str", "list", "dict", "123", 123]:
```

Comment:
Im not sure this test should be in place. This is testing a usage that CANNOT happen if the user uses the library as intended. We have a loader and validator in place that will not allow calling the test_schema_section method with this type of data. We therefore do not need to test it or handle this case in the code, and doing so (as was before) is over engineering that makes our lives harder.

Reply:
Yeah I don't disagree with this. I'll remove the test.